schemas/fluidity_options.rnc

```diff
@@ -3562,12 +3662,106 @@ prognostic_detector_options = detector_options_enabled_default
 diagnostic_detector_options = detector_options_enabled_default
 prescribed_detector_options = detector_options_disabled_default
 
+# Specify whether or not values for this field are interpolated to particle locations.
+particle_options_enabled_default = 
+   (
+      ## Specify what is added to particles
+      element particles {
+         (
+            ## This field is included at each particle location.
+            element include_in_particles {
+	       element store_old_field {
+	         ## Store values of field at preivous timestep/non-linear iteration, for use when updating particle attributes.
+	       	  comment
+	       }?
+            }|
+            ## Field value is not interpolated to particle location.
+            element exclude_from_particles {
+	      ## Field values are not stored on particles.
+               comment
+            }
+         )
+      }
+   )
+
+# Options for whether a field is to be included in particles.
+particle_options_disabled_default = 
```

Comment:
This and `particle_options_enabled_default` should re-use definitions of exclude and include in_particles.  If that's not possible for some reason then same comments about docstrings here too.